src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -24,18 +24,11 @@ $spacing: 20px;
   &__main {
     display: flex;
     flex-direction: row;
+    flex: auto 1;
```

Comment:
What was the intent behind adding this here?

Reply:
Hm, I thought it did something but it seems like it doesn't- I'll delete it